### PR TITLE
Migration Disabled - Withdraw Enabled

### PR DESCRIPTION
**Pull request by RanCohenn**
added #695 #696 #701

### CODE REVIEW

File: src/services/web3/protection/positions.ts [hunk 1]
```diff
@@ -280,6 +281,10 @@ export const fetchProtectedPositions = async (
     );
 
     const vaultBalance = 70;
+    const deficit = {
+      tkn: '70',
```
Comment (zavelevsky) [inline src/services/web3/protection/positions.ts:285]: what's the meaning of the hardcoded values? (70)

**Comment by Yanivshechter1** [comment]
@RanCohenn 
Please remove all text and places related to protected positions on the v2.1 portfolio screen 
1. Remove the sentence - "Manage your protected positions in Bancor pools and track and analyze your returns."
2. Under my stake remove "Protected Value" section 
3. Change the title: "My Protected Positions" to "My Positions" 
4. Remove "Closed Positions" section from the screen

**Comment by Yanivshechter1** [comment]
@RanCohenn 
On the withdrawal widget, please remove 
1. Vault balance section 
2. text - "Depending on the state of the liquidity pool being withdrawn from, positional impermanent loss in addition to a pro rata loss may be experienced upon withdrawal. Migration of user funds from v2.1 to v3 will remain permanently disabled."

**Comment by Yanivshechter1** [comment]
@RanCohenn Please change the pop-up text to the new version - 
"Direct withdrawals from v2.1 are now open. Depending on the state of the liquidity pool being withdrawn from, positional impermanent loss in addition to a pro rata loss may be experienced upon withdrawal. Migration of user funds from v2.1 to v3 will remain permanently disabled."

**Comment by Yanivshechter1** [comment]
@RanCohenn 
When inserting a bigger amount then the available amount 
error "insufficient balance" is too close to the checkbox text

**Comment by Yanivshechter1** [comment]
@RanCohenn Withdraw BNT does not work- Withdraw button is always disabled

**Review by zavelevsky — DISMISSED** [review]
LGTM

**Comment by Yanivshechter1** [comment]
@RanCohenn 
**Withdraw amount changes** - We no longer support partial withdrawal and from now on user must withdraw all  therefore, we will have to make the following changes:
1. Remove the available amount
2. Use the "removeLiquidityReturn" to automatically set the amount the user will withdraw 
3. Remove the ability the edit the input field 
4. Always pass to the transaction portion=100 (It is in ppm so 1000000)
User flow:
User clicks withdraw -> withdraw popup is presented with the full amount already inserted in the input field -> User approves the BNT disclaimer -> Withdraw CTA button is enabled

**Review by ashachaf — DISMISSED** [review]

**Pull request merged**